vault/expiration.go

```diff
@@ -652,11 +687,25 @@ func (m *ExpirationManager) revokePrefixCommon(prefix string, force bool) error
 	// Revoke all the keys
 	for idx, suffix := range existing {
 		leaseID := prefix + suffix
-		if err := m.revokeCommon(leaseID, force, false); err != nil {
-			return errwrap.Wrapf(fmt.Sprintf("failed to revoke %q (%d / %d): {{err}}", leaseID, idx+1, len(existing)), err)
+		switch sync {
```

Comment:
Can we use an if/else here instead since `sync` is a bool?

Reply:
Done.